pages/category/index.vue

```diff
@@ -1,3 +1,847 @@
+  }
+  &__aside {
+    padding: 0;
+  }
+  &__main {
+    flex: 1;
+    display: flex;
+    padding: 0;
+    @include for-desktop {
+      padding: var(--spacer-xs) var(--spacer-xl);
+    }
+  }
+  &__title {
+    --heading-title-font-weight: var(--font-weight--semibold);
+    --heading-title-font-size: var(--h3-font-size);
+  }
+  &__filters-icon {
+    margin: 0 0 0 var(--spacer-2xs);
+    order: 1;
+    @include for-desktop {
+      margin: 0 var(--spacer-xs) 0 0;
+      order: 0;
+    }
+  }
+  &__plus-icon {
+    margin: 0 0 0 var(--spacer-2xs);
+    order: 1;
+    @include for-desktop {
+      margin: 0 var(--spacer-xs) 0 0;
+      order: 0;
+    }
+  }
+  &__filters-button {
+    display: flex;
+    align-items: center;
+
+    --button-font-size: var(--font-size--base);
+    --button-text-decoration: none;
+    --button-color: var(--c-link);
+    --button-font-weight: var(--font-weight--normal);
+    @include for-mobile {
+      --button-font-weight: var(--font-weight--medium);
+
+      margin-right: var(--spacer-sm);
+      order: 1;
+    }
+    svg {
+      fill: var(--c-text-muted);
+      transition: fill 150ms ease;
+    }
+    &:hover {
+      svg {
+        fill: var(--c-primary);
+      }
+    }
+  }
+  &__plus-button {
+    display: flex;
+    align-items: center;
+    margin: var(--spacer-sm) 0;
+
+    --button-font-size: var(--font-size--sm);
+    --button-text-decoration: none;
+    --button-color: var(--c-link);
+    --button-font-weight: 100;
+    @include for-mobile {
+      --button-font-weight: var(--font-weight--medium);
+
+      margin-right: var(--spacer-sm);
+      order: 1;
+    }
+  }
+  &__back-button {
+    display: flex;
+    align-items: center;
+    margin: var(--spacer-sm) 0 0 0;
+
+    --button-font-size: var(--font-size--sm);
+    --button-text-decoration: none;
+    --button-color: var(--c-link);
+    --button-font-weight: var(--font-weight--sm);
+    @include for-mobile {
+      --button-font-weight: var(--font-weight--medium);
+
+      margin-right: var(--spacer-sm);
+      order: 1;
+    }
+  }
+  &__label {
+    font-family: var(--font-family--secondary);
+    font-weight: var(--font-weight--normal);
+    color: var(--c-text-muted);
+    @include for-desktop {
+      margin: 0 var(--spacer-2xs) 0 0;
+    }
+  }
+  &__select {
+    --component-select-width: 220px;
+    --component-select-padding: 0;
+    --component-select-selected-padding: 0 var(--spacer-lg) 0 var(--spacer-2xs);
+    --component-select-margin: 0;
+    --component-select-error-message-height: 0;
+  }
+  &__sort {
+    display: flex;
+    align-items: center;
+    margin: 0 0 0 auto;
+  }
+  &__counter {
+    font-family: var(--font-family--secondary);
+    margin: auto;
+    @include for-desktop {
+      margin: auto 0 auto auto;
+    }
+  }
+  &__view {
+    display: flex;
+    order: -1;
+    align-items: center;
+    margin: 0;
+    @include for-desktop {
+      margin: 0 0 0 var(--spacer-2xl);
+      order: 0;
+    }
+    &-icon {
```

Comment:
As per the BEM standard &-icon is not right convention, I think it should be &__icon

Reply:
Resolved